changelog/unreleased/pull-2840

```diff
@@ -0,0 +1,6 @@
+Enhancement: Parallel file deletion in forget, prune and rebuild-index
+
+We've added parallel file deletion for the commands forget, prune and rebuild-index.
```

Comment:
I'd swap these two sentences such that the explanation mentions the change first and then provides a hint that this is achieved by deleting multiple files in parallel.

Reply:
changed.